client/App.tsx

```diff
@@ -26,10 +29,41 @@ function App() {
     <div>
       <h1>Welcome to Grats + Relay</h1>
       Server says: "{data.greeting}"
+      <ErrorBoundary fallback={<Fallback />}>
+        <ChildComponent query={data} />
+      </ErrorBoundary>
     </div>
   );
 }
 
+function Fallback() {
+  return (
+    <span style={{ border: "1px solid pink", padding: "10px" }}>
+      Oops! Something went wrong
+    </span>
+  );
+}
+
+function ChildComponent(props: { query: AppChildComponent$key }) {
+  const data = useFragment(
+    graphql`
+      fragment AppChildComponent on Query {
+        # When this errors, the component will throw and
+        # the error will be caught by the ErrorBoundary
+        alwaysThrows
```

Comment:
When this field errors, it is contained by the surrounding ErrorBoundary on line 32